### PR TITLE
feat: implement conditional ticket masking following passwordmasking.com's recs

### PR DESCRIPTION
**Pull request by shrugs**
well that was straightforward. love final-form now that it works (lol)

### CODE REVIEW

**Comment by shrugs** [comment]
wait I forgot to bring this behavior to the other palces we use ticket input like actiavtecode and ticket validation. 1 moment

**Comment by shrugs** [comment]
@Fang- ok fixed

File: src/views/Login/Ticket.js [hunk 1]
```diff
@@ -135,6 +137,8 @@ export default function Ticket({ className, goHome }) {
       point: impliedPoint || '',
       usePasshrase: false,
       useShards: false,
+      showTicket: true,
```
Comment (Fang-) [inline src/views/Login/Ticket.js]: wait, I missed this, but shouldn't this be default `false`, as per common password field convention?

Comment (shrugs) [reply src/views/Login/Ticket.js]: the website mentions that:

> When offered the choice of masked or unmasked passwords within the interface, the concept was identified as a feature not an error. Participants appreciated the usability benefits of clear text passwords and the security benefits of masked passwords. When participants used this option their trustingness of the website was unaffected. (Note the password was unmasked by default)

Comment (Fang-) [reply src/views/Login/Ticket.js]: Oh I hadn't remembered that note at the end. Very interesting. This is probably fine then.

**Pull request merged**